code-of-conduct.md

```diff
@@ -1 +1,92 @@
 # Triangle DevOps Code of Conduct
+
+Triangle DevOps is dedicated to providing a harassment-free community experience
+for everyone, regardless of gender, gender identity and expression, sexual
+orientation, disability, physical appearance, body size, race, age or religion.
+We do not tolerate harassment of community participants in any form.
+
+Community participants and organizers violating these rules may be sanctioned or
+expelled from the community at the discretion of the organizing team.
+
+Harassment includes, but is not limited to:
+
+* Verbal comments that reinforce social structures of domination related to
+  gender, gender identity and expression, sexual orientation, disability,
+  physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+If you are being harassed, notice someone else is being harassed, or have other
+concerns, please contact an organizer immediately.
+
+## Enforcement
+
+Participants asked to stop any harassing behavior are expected to comply
+immediately. If a participant engages in harassing behaviour, the organisers
+retain the right to take any actions to keep events and online spaces as
+welcoming environments for all participants.
+
+Action may include warning the offender. Action could also include expelling the
+offender from meetings, events, and any online spaces this group uses on a
+temporary or permanent basis. A warning is not required to expel an offender.
+
+The organizers will seek to use the least punitive means available to resolve an
+issue. If the circumstances require asking an offender to leave, we will do
+that.
+
+These rules apply at all meetings, events, meeting or event-related social
+activities (e.g. a meal or beer before or after the main event). These rules
+also apply to the group's online spaces.
+
+## Reporting
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it
+as soon as possible. Harassment and other code of conduct violations reduce the
+value of our events and online spaces for everyone. We want you to be happy at
+our meetings and events and participating in our online spaces. People like you
+make them all better.
+
+* Talking to the on-site organizers or organizers, who will introduce themselves
+  at the beginning any event
+* Emailing the organizing team [FIXME: email address for contact]
```

Comment:
I've reached out to Meetup. Waiting to hear what they respond with. On Twitter they suggested having a group email wasn't something they normally provide, but that they might have a workaround and that I should reach out to them as I've done.